ccdproc/core.py

```diff
@@ -1173,30 +1198,37 @@ def cosmicray_lacosmic(ccd, sigclip=4.5, sigfrac=0.3,
         the mask. Default: 65536.0.
     niter : int, optional
         Number of iterations of the LA Cosmic algorithm to perform. Default: 4.
-    sepmed : boolean, optional
+    sepmed : bool, optional
         Use the separable median filter instead of the full median filter.
         The separable median is not identical to the full median filter, but
         they are approximately the same and the separable median filter is
         significantly faster and still detects cosmic rays well. Default: True
     cleantype : {'median', 'medmask', 'meanmask', 'idw'}, optional
-        Set which clean algorithm is used:\n
-        'median': An umasked 5x5 median filter\n
-        'medmask': A masked 5x5 median filter\n
-        'meanmask': A masked 5x5 mean filter\n
-        'idw': A masked 5x5 inverse distance weighted interpolation\n
+        Set which clean algorithm is used:
+
+        - 'median': An umasked 5x5 median filter
```

Comment:
![unbenannt](https://cloud.githubusercontent.com/assets/14200878/14623597/ef42f9fe-05d3-11e6-88ee-639ae857585b.jpg)

Maybe I should put some of the options in \`\` to highlight them. I'm not sure. But rendering looks ok.